backend/requirements.txt

```diff
@@ -10,16 +11,22 @@ click==8.1.7
 dataclasses-json==0.6.7
 dnspython==2.6.1
 email_validator==2.2.0
-fastapi==0.109.0
+fastapi==0.115.6
```

Comment:
Consider upgrading fastapi from 0.109.0 to 0.115.6 for better performance and security.